cmd/util/ledger/migrations/migrator_runtime.go

```diff
@@ -31,35 +41,54 @@ func newMigratorRuntime(
 	if err != nil {
 		return nil, fmt.Errorf("failed to create payload snapshot: %w", err)
 	}
+
 	transactionState := state.NewTransactionState(snapshot, state.DefaultParameters())
 	accounts := environment.NewAccounts(transactionState)
 
 	accountsAtreeLedger := util.NewAccountsAtreeLedger(accounts)
-	storage := runtime.NewStorage(accountsAtreeLedger, util.NopMemoryGauge{})
+	runtimeStorage := runtime.NewStorage(accountsAtreeLedger, util.NopMemoryGauge{})
+
+	derivedChainData, err := derived.NewDerivedChainData(derived.DefaultDerivedDataCacheSize)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create derived chain data: %w", err)
+	}
+
+	// TODO: how to get currentBlockId (flow.Identifier) for NewDerivedBlockDataForScript?
```

Comment:
The current block does not matter here. That Is only for keeping a cross-block cache, which is not needed here.

Reply:
```suggestion
	// The current block ID does not matter here, it is only for keeping a cross-block cache, which is not needed here.
```